metadata/params/params.go

```diff
@@ -71,7 +105,22 @@ func visitAllParams(components ast.Object) (map[string]Params, error) {
 		if err != nil {
 			return nil, err
 		}
-		params[string(*f.Id)] = p
+		var id string
+		switch f.Kind {
+		case ast.ObjectFieldStr:
+			c, _ := f.Expr1.(*ast.LiteralString)
+			switch c.Kind {
+			case ast.StringSingle:
+				fallthrough
+			case ast.StringDouble:
```

Comment:
Mentioned offline, but plan is to use multi-case syntax, like: `case ast.StringSingle, ast.StringDouble:`. `fallthrough` is fraught.